content/fr/open-source.md

```diff
@@ -72,35 +98,35 @@ Voici un aperçu des principaux dépôts que nous avons créés directement ou i
 </article>
 
 <article class="list-article icon-list-article">
-    <div class="col25 article-featured-image"><a href="https://github.com/baltpeter/yace"><img class="image" src="/card-icons/speech-bubble.svg" alt="yace"></a></div>
+    <div class="col25 article-featured-image"><a href="https://github.com/datenanfragen/backend"><img class="image" src="/card-icons/speech-bubble.svg" alt="backend"></a></div>
     <div class="padded col75">
-        <a href="https://github.com/baltpeter/yace"><h1><code>yace</code></h1></a>
+        <a href="https://github.com/datenanfragen/backend"><h1><code>backend</code></h1></a>
         <span class="license">Licence MIT</span>
         <p class="description">
-            Un moteur simple, respectueux de la vie privée et facile à déployer pour créer des solutions de commentaires personnalisées, hébergé sur AWS.
+            Un serveur de développement qui définit les paramètres des fonctionnalités dynamiques du site internet comme les commentaires, les suggestions eet les donations.:;;
         </p>
     </div>
     <div class="clearfix"></div>
-    <a class="button button-primary read-more-button" href="https://github.com/baltpeter/yace">Voir le dépôt&nbsp;<span class="icon icon-arrow-right"></span></a>
+    <a class="button button-primary read-more-button" href="https://github.com/datenanfragen/backend">Voir le dépôt&nbsp;<span class="icon icon-arrow-right"></span></a>
 </article>
 
 <a id="contributeurs"></a>
 ## Contributeurs
 
-Un grand merci à tous les contributeurs de demandetesdonnees.fr ! Le projet ne serait pas possible sans vous.
+Un grand merci à tous les contributeurs(trices) de demandetesdonnees.fr! Le projet ne serait pas possible sans vous.
```

Comment:
There is a missing space before exclamation mark

Reply:
done! :)